File: aula_11-04/3_adivinhação.py
# Este código tenta implementar um jogo simples de adivinhação de números.
# O programa deve escolher um número aleatório entre 1 e 10 e permitir que o usuário tente adivinhar o número.
# No entanto, o código está incompleto e contém erros.
# Complete e corrija o código para que o jogo funcione corretamente.
# O usuário tem três tentativas para adivinhar o número. Após cada tentativa, o programa deve informar
# se o palpite é muito alto, muito baixo ou correto.

import random

numero_secreto = random.randint(1, 10)
tentativas = 3

while tentativas > 0:
    palpite = int(input("Adivinhe o número entre 1 e 10: "))
    if palpite == numero_secreto:
        print("Parabéns! Você acertou.")
        break
    tentativas -= 1

print("Fim do jogo.")

File: aula_12-04/3_adivinhação.cpp
#include <iostream>
#include <cstdlib>  // Necessário para rand() e srand()
#include <ctime>    // Necessário para time()

using namespace std;

int main() {
    srand(time(0)); // Inicializa o gerador de números aleatórios
    int numero_secreto = rand() % 10 + 1;
    int tentativas = 3;
    int palpite;

    while (tentativas > 0) {
        cout << "Adivinhe o número entre 1 e 10: ";
        cin >> palpite;

        if (palpite == numero_secreto) {
            cout << "Parabéns! Você acertou." << endl;
            break;
        }
        tentativas--;
    }

    return 0;
}
